docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -487,22 +496,67 @@ export default function AppLayoutDocsFooter(props) {
                   {t('submit')}
                 </Button>
               </DialogActions>
-              {rating !== 1 && (
-                <Typography id="feedback-description" color="text.secondary">
-                  {t('feedbackMessageToGitHub.usecases')}
-                  <br />
-                  {t('feedbackMessageToGitHub.callToAction.text')}
+            </Box>
+          </form>
+        </Collapse>
+        <Divider sx={{ mb: 4 }} />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', md: 'row' },
+            justifyContent: 'space-between',
+            gap: { xs: 4, lg: 12 },
+          }}
+        >
+          <Box flexGrow={1}>
+            <Typography variant="body2" fontWeight="bold">
+              Keep up to date
+            </Typography>
+            <Typography variant="body2" color="text.secondary" sx={{ mb: 2 }}>
+              Join our newsletter for regular updates. No spam ever.
+            </Typography>
+            <EmailSubscribe fullWidth sx={{ mb: 1 }} />
```

Comment:
I think that it would be great to have analytics on the source of the people that filled it. I assume this can be done by creating a new form in Sendinblue.

I find it a bit strange to have the newsletter link on the docs, as a developer, I don't see why I would subscribe in this context. On the blog pages, it makes a lot of sense to me.

Reply:
Re analytics: for sure, that's a great idea 👍 

Re newsletter: I think it's a cheap/non-disruptive additional entry point to the newsletter. As we make email a more robust means of communication, adding more opportunities to capture people's contact doesn't seem to hurt...

Reply:
> to capture people's contact doesn't seem to hurt...

My main concern is that it feels dense, the design catches my attention x2 more than it does on https://stripe.com/docs/no-code/pricing-table, I assume away from more important information.

Reply:
Gotcha ⎯ I've changed that! 👍